lib/spack/spack/test/cmd/env.py

```diff
@@ -966,12 +990,16 @@ def test_included_config_precedence(environment_from_manifest):
 """
         )
 
+    e = ev.Environment(tmp_path)
     with e:
         e.concretize()
```

Comment:
Same thing here. Just need to standardize on a single way of calling `concretize()`.

Reply:
Same here about not going through the overhead of the concretize command.